LICENSE.md

```diff
@@ -93,21 +90,21 @@ A Derivative must contain changes which a reasonably informed person would consi
 The User may distribute a Derivative if the following conditions are met:
  - The Derivative does not generate revenue.
  - The Derivative provides credit to the Author.
- - The Derivative complies with the Mod license.
+ - The Derivative, as a whole, is licensed under this license for anyone who comes into possession of a copy.
+ - The User provides means to obtain the Derivative's source code at no charge.
 
-8. Right to distribute addons
+7. Right to distribute addons
 --------
 
 The User may develop non-Derivative Addons.  
 An Addon may include the Mod API without being considered a Derivative.  
 The Addon may be distributed under a different license.  
 
-9. Modpack rights
+8. Modpack rights
 --------
 
 The User acknowledges that a Modpack will make changes to the Mod, and the Mod or Game may not work.  
 The Author may not be held responsible for the incompetency or sadism of the Modpack Creator.  
 
-Modpacks may not be used to generate revenue unless specifically granted this right by the Author.  
-
-Modpacks may be made and distributed by the Modpack Creator through any distribution channel they choose.  
+The distribution of this Mod as part of a modpack is subject to section 4. Notwithstanding, modpacks containing this Mod may be made and distributed by the Modpack Creator through any distribution channel they choose.  
+Modpacks containing this Mod may not be used to generate revenue unless specifically granted this right by the Author.  
```

Comment:
This was stated better in the line you removed. Don't convolute things for the sake of making it sound like legalese.